querybook/server/lib/result_store/stores/file_store.py

```diff
@@ -27,25 +27,43 @@ def __init__(self, uri: str):
         self.uri = get_file_uri(uri)
 
     def start(self):
+        self._chunk_size=100000
         self._chunks_length = 0
+        self._chunks = []
         os.makedirs(self.uri_dir_path, exist_ok=True)
 
+    def _reset_variables(self):
+        self._chunks = []
+        self._chunks_length = 0
```

Comment:
also `_chunks_length` should not be reset as 0, as it is used to compare with `QuerybookSettings.DB_MAX_UPLOAD_SIZE` , it needs to be the total length.

Reply:
updated